apps/src/templates/progress/progressHelpers.js

```diff
@@ -75,12 +74,8 @@ export function stageLocked(levels) {
  * @returns A friendly name for the icon name (that can be passed to FontAwesome)
  *   for the given level.
  */
-export function getIconForLevel(level, inDetailedProgressView = false) {
```

Comment:
@madelynkasula why lose the inDetailedProgressView here? Is that causing performance issues as well?

Reply:
the variable name is not quite accurate -- `inMiniRubricExperiment` really means "in experiment and in details progress view" here. Perhaps it can be renamed to `showAssessmentChecks` in a future PR.

Reply:
Oh true! Ok awesome:)